GNOME: update extensions for v46

msfjarvis opened the pull request:
## Description of changes

Updates the GNOME extensions set in its entirety to bring in support for GNOME 46.

I've updated the `extensions.json` file with the `update-extensions.py` script, and `nix-update`'d the ones we build directly if there were compatible versions available.

Opened as a separate PR as requested here: https://github.com/NixOS/nixpkgs/pull/291339#issuecomment-2045420740

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

msfjarvis committed:
gnomeExtensions: add support for GNOME 46

msfjarvis committed:
gnomeExtensions: pull in updates for GNOME 46

msfjarvis committed:
gnomeExtensions.arcmenu: 52 -> 55

Diff: https://gitlab.com/arcmenu/ArcMenu/-/compare/v52...v55

msfjarvis committed:
gnomeExtensions.pop-shell: set `passthru.updateScript`

msfjarvis committed:
gnomeExtensions.pop-shell: unstable-2023-11-10 -> unstable-2024-04-04

msfjarvis committed:
gnomeExtensions.mullvad-indicator: add patch for GNOME 46 compat

msfjarvis committed:
gnomeExtensions.argos: set `passthru.updateScript`

msfjarvis committed:
gnomeExtensions.argos: unstable-2023-09-26 -> unstable-2024-04-03

msfjarvis commented:
Result of `nixpkgs-review pr 302860` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>308 packages built:</summary>
  <ul>
    <li>gnomeExtensions.activate-window-by-title</li>
    <li>gnomeExtensions.activate_gnome</li>
    <li>gnomeExtensions.add-custom-text-to-workspace-indicators</li>
    <li>gnomeExtensions.add-username-to-top-panel</li>
    <li>gnomeExtensions.advanced-alttab-window-switcher</li>
    <li>gnomeExtensions.allow-locked-remote-desktop</li>
    <li>gnomeExtensions.alphabetical-app-grid</li>
    <li>gnomeExtensions.alttab-launcher</li>
    <li>gnomeExtensions.always-show-workspace-thumbnails</li>
    <li>gnomeExtensions.app-hider</li>
    <li>gnomeExtensions.app-menu-is-back</li>
    <li>gnomeExtensions.applications-menu</li>
    <li>gnomeExtensions.applications-overview-tooltip</li>
    <li>gnomeExtensions.arch-linux-updates-and-security-indicator</li>
    <li>gnomeExtensions.archlinux-updates-indicator</li>
    <li>gnomeExtensions.arcmenu</li>
    <li>gnomeExtensions.argos</li>
    <li>gnomeExtensions.arrange-windows</li>
    <li>gnomeExtensions.astra-monitor</li>
    <li>gnomeExtensions.astrapios-panel-menu</li>
    <li>gnomeExtensions.auto-activities</li>
    <li>gnomeExtensions.auto-move-windows</li>
    <li>gnomeExtensions.auto-power-profile</li>
    <li>gnomeExtensions.autohide-battery</li>
    <li>gnomeExtensions.awesome-tiles</li>
    <li>gnomeExtensions.battery-indicator-icon</li>
    <li>gnomeExtensions.battery-usage-wattmeter</li>
    <li>gnomeExtensions.better-end-session-dialog</li>
    <li>gnomeExtensions.bing-wallpaper-changer</li>
    <li>gnomeExtensions.bluetooth-battery</li>
    <li>gnomeExtensions.bluetooth-quick-connect</li>
    <li>gnomeExtensions.blur-my-shell</li>
    <li>gnomeExtensions.bottom-overview</li>
    <li>gnomeExtensions.bottom-panel</li>
    <li>gnomeExtensions.bring-out-submenu-of-power-offlogout-button</li>
    <li>gnomeExtensions.bubblemail</li>
    <li>gnomeExtensions.caffeine</li>
    <li>gnomeExtensions.calc</li>
    <li>gnomeExtensions.chronos-time-tracker</li>
    <li>gnomeExtensions.click-to-close-overview</li>
    <li>gnomeExtensions.clipboard-history</li>
    <li>gnomeExtensions.clipboard-indicator</li>
    <li>gnomeExtensions.cloudflare-warp-indicator</li>
    <li>gnomeExtensions.cloudflare-warp-toggle</li>
    <li>gnomeExtensions.clown</li>
    <li>gnomeExtensions.color-picker</li>
    <li>gnomeExtensions.colorblind-filters</li>
    <li>gnomeExtensions.colortint</li>
    <li>gnomeExtensions.compiz-alike-magic-lamp-effect</li>
    <li>gnomeExtensions.compiz-windows-effect</li>
    <li>gnomeExtensions.control-blur-effect-on-lock-screen</li>
    <li>gnomeExtensions.copier</li>
    <li>gnomeExtensions.coverflow-alt-tab</li>
    <li>gnomeExtensions.custom-accent-colors</li>
    <li>gnomeExtensions.custom-hot-corners-extended</li>
    <li>gnomeExtensions.custom-osd</li>
    <li>gnomeExtensions.custom-window-controls</li>
    <li>gnomeExtensions.customize-clock-on-lock-screen</li>
    <li>gnomeExtensions.customize-ibus</li>
    <li>gnomeExtensions.dash-to-dock-workaround</li>
    <li>gnomeExtensions.dash-to-panel</li>
    <li>gnomeExtensions.dash2dock-lite</li>
    <li>gnomeExtensions.dashbar</li>
    <li>gnomeExtensions.date-menu-formatter</li>
    <li>gnomeExtensions.ddterm</li>
    <li>gnomeExtensions.debian-linux-updates-indicator</li>
    <li>gnomeExtensions.default-workspace</li>
    <li>gnomeExtensions.dejaview</li>
    <li>gnomeExtensions.desktop-lyric</li>
    <li>gnomeExtensions.dim-completed-calendar-events</li>
    <li>gnomeExtensions.disable-extension-updates</li>
    <li>gnomeExtensions.disable-gestures-2021</li>
    <li>gnomeExtensions.disable-menu-switching</li>
    <li>gnomeExtensions.disable-workspace-animation</li>
    <li>gnomeExtensions.disable-workspace-switch-animation-for-gnome-40</li>
    <li>gnomeExtensions.disable-workspace-switcher-overlay</li>
    <li>gnomeExtensions.display-scale-switcher</li>
    <li>gnomeExtensions.do-not-disturb-while-screen-sharing-or-recording</li>
    <li>gnomeExtensions.dock-from-dash</li>
    <li>gnomeExtensions.dock-reloaded</li>
    <li>gnomeExtensions.downfall</li>
    <li>gnomeExtensions.easy-docker-containers</li>
    <li>gnomeExtensions.easyeffects-preset-selector</li>
    <li>gnomeExtensions.emoji-copy</li>
    <li>gnomeExtensions.enable-debug-logs</li>
    <li>gnomeExtensions.end-session-timer</li>
    <li>gnomeExtensions.esp-extensions-search-provider</li>
    <li>gnomeExtensions.executor</li>
    <li>gnomeExtensions.extension-list</li>
    <li>gnomeExtensions.fancy-tiling</li>
    <li>gnomeExtensions.fedora-linux-update-indicator</li>
    <li>gnomeExtensions.fixed-ime-list</li>
    <li>gnomeExtensions.floating-dock-2</li>
    <li>gnomeExtensions.focus-changer</li>
    <li>gnomeExtensions.focused-window-d-bus</li>
    <li>gnomeExtensions.force-quit</li>
    <li>gnomeExtensions.forge</li>
    <li>gnomeExtensions.freon</li>
    <li>gnomeExtensions.frequency-boost-switch</li>
    <li>gnomeExtensions.frippery-applications-menu</li>
    <li>gnomeExtensions.fullscreen-avoider</li>
    <li>gnomeExtensions.fullscreen-hot-corner</li>
    <li>gnomeExtensions.fullscreen-notifications</li>
    <li>gnomeExtensions.gbinaryclock</li>
    <li>gnomeExtensions.gemini-ai</li>
    <li>gnomeExtensions.github-actions</li>
    <li>gnomeExtensions.gnome-40-ui-improvements</li>
    <li>gnomeExtensions.gnome-bedtime</li>
    <li>gnomeExtensions.gnordvpn-local</li>
    <li>gnomeExtensions.grand-theft-focus</li>
    <li>gnomeExtensions.gtile</li>
    <li>gnomeExtensions.gtk4-desktop-icons-ng-ding</li>
    <li>gnomeExtensions.haguichi-indicator</li>
    <li>gnomeExtensions.happy-appy-hotkey</li>
    <li>gnomeExtensions.headsetcontrol</li>
    <li>gnomeExtensions.hibernate-status-button</li>
    <li>gnomeExtensions.hide-activities-button</li>
    <li>gnomeExtensions.hide-cursor</li>
    <li>gnomeExtensions.hide-items</li>
    <li>gnomeExtensions.hide-keyboard-layout</li>
    <li>gnomeExtensions.hide-minimized</li>
    <li>gnomeExtensions.hide-the-dock-in-overview</li>
    <li>gnomeExtensions.hide-top-bar</li>
    <li>gnomeExtensions.hide-universal-access</li>
    <li>gnomeExtensions.home-assistant-extension</li>
    <li>gnomeExtensions.hot-edge</li>
    <li>gnomeExtensions.ibus-switcher</li>
    <li>gnomeExtensions.ibus-tweaker</li>
    <li>gnomeExtensions.icinga-checker</li>
    <li>gnomeExtensions.icon-hider-updated</li>
    <li>gnomeExtensions.ideapad</li>
    <li>gnomeExtensions.impatience</li>
    <li>gnomeExtensions.indicator</li>
    <li>gnomeExtensions.inhibition-indicator</li>
    <li>gnomeExtensions.input-source-d-bus-interface</li>
    <li>gnomeExtensions.ip-finder</li>
    <li>gnomeExtensions.just-another-search-bar</li>
    <li>gnomeExtensions.just-perfection</li>
    <li>gnomeExtensions.kerberos-login</li>
    <li>gnomeExtensions.keyring-autolock</li>
    <li>gnomeExtensions.kimpanel</li>
    <li>gnomeExtensions.krypto</li>
    <li>gnomeExtensions.lan-ip-address</li>
    <li>gnomeExtensions.launch-new-instance</li>
    <li>gnomeExtensions.launcher</li>
    <li>gnomeExtensions.layout-hotkeys</li>
    <li>gnomeExtensions.legacy-gtk3-theme-scheme-auto-switcher</li>
    <li>gnomeExtensions.light-dict</li>
    <li>gnomeExtensions.light-style</li>
    <li>gnomeExtensions.lineup</li>
    <li>gnomeExtensions.live-captions-assistant</li>
    <li>gnomeExtensions.lock-keys</li>
    <li>gnomeExtensions.luminus-desktop</li>
    <li>gnomeExtensions.luminus-shell</li>
    <li>gnomeExtensions.luminus-shell-y</li>
    <li>gnomeExtensions.lunar-calendar</li>
    <li>gnomeExtensions.memento-mori</li>
    <li>gnomeExtensions.messaging-menu</li>
    <li>gnomeExtensions.middle-click-to-close-in-overview</li>
    <li>gnomeExtensions.miniview</li>
    <li>gnomeExtensions.move-clock</li>
    <li>gnomeExtensions.mpris-label</li>
    <li>gnomeExtensions.mullvad-indicator</li>
    <li>gnomeExtensions.mute-spotify-ads</li>
    <li>gnomeExtensions.muteunmute</li>
    <li>gnomeExtensions.mypubip</li>
    <li>gnomeExtensions.native-window-placement</li>
    <li>gnomeExtensions.net-speed</li>
    <li>gnomeExtensions.net-speed-simplified</li>
    <li>gnomeExtensions.new-workspace-shortcut</li>
    <li>gnomeExtensions.night-theme-switcher</li>
    <li>gnomeExtensions.no-a11y</li>
    <li>gnomeExtensions.no-overview</li>
    <li>gnomeExtensions.no-titlebar-when-maximized</li>
    <li>gnomeExtensions.nothing-to-say</li>
    <li>gnomeExtensions.notification-banner-reloaded</li>
    <li>gnomeExtensions.notification-timeout</li>
    <li>gnomeExtensions.on-the-top</li>
    <li>gnomeExtensions.open-bar</li>
    <li>gnomeExtensions.open-browser-tabs-on-active-workspace</li>
    <li>gnomeExtensions.osd-volume-number</li>
    <li>gnomeExtensions.otp-keys</li>
    <li>gnomeExtensions.panel-corners</li>
    <li>gnomeExtensions.panel-date-format</li>
    <li>gnomeExtensions.panel-favorites</li>
    <li>gnomeExtensions.panel-free</li>
    <li>gnomeExtensions.panel-note</li>
    <li>gnomeExtensions.panel-scroll</li>
    <li>gnomeExtensions.panel-workspace-scroll</li>
    <li>gnomeExtensions.panel-world-clock-lite</li>
    <li>gnomeExtensions.paperwm</li>
    <li>gnomeExtensions.persian-calendar</li>
    <li>gnomeExtensions.persian-calendar-2</li>
    <li>gnomeExtensions.pi-hole</li>
    <li>gnomeExtensions.picture-of-the-day</li>
    <li>gnomeExtensions.places-status-indicator</li>
    <li>gnomeExtensions.pop-shell</li>
    <li>gnomeExtensions.power-profile-indicator</li>
    <li>gnomeExtensions.power-profile-switcher</li>
    <li>gnomeExtensions.primary-input-on-lockscreen</li>
    <li>gnomeExtensions.proton-bridge-button</li>
    <li>gnomeExtensions.proton-vpn-button</li>
    <li>gnomeExtensions.proxy-switcher</li>
    <li>gnomeExtensions.quake-terminal</li>
    <li>gnomeExtensions.quick-settings-audio-devices-hider</li>
    <li>gnomeExtensions.quick-settings-audio-devices-renamer</li>
    <li>gnomeExtensions.quick-settings-audio-panel</li>
    <li>gnomeExtensions.quick-text</li>
    <li>gnomeExtensions.quick-touchpad-toggle</li>
    <li>gnomeExtensions.rdesktop-launcher</li>
    <li>gnomeExtensions.reading-strip</li>
    <li>gnomeExtensions.reboottouefi</li>
    <li>gnomeExtensions.rectangle</li>
    <li>gnomeExtensions.remmina-search-provider</li>
    <li>gnomeExtensions.removable-drive-menu</li>
    <li>gnomeExtensions.reorder-workspaces</li>
    <li>gnomeExtensions.run-or-raise</li>
    <li>gnomeExtensions.runcat</li>
    <li>gnomeExtensions.sane-airplane-mode</li>
    <li>gnomeExtensions.screen-rotate</li>
    <li>gnomeExtensions.screen-word-translate</li>
    <li>gnomeExtensions.screenshort-cut</li>
    <li>gnomeExtensions.screenshot-window-sizer</li>
    <li>gnomeExtensions.search-light</li>
    <li>gnomeExtensions.settingscenter</li>
    <li>gnomeExtensions.shortcuts</li>
    <li>gnomeExtensions.shortcuts-to-activate-input-methods</li>
    <li>gnomeExtensions.show-favorite-apps</li>
    <li>gnomeExtensions.shu-zhi</li>
    <li>gnomeExtensions.shutdowntimer</li>
    <li>gnomeExtensions.simple-break-reminder</li>
    <li>gnomeExtensions.simple-net-speed</li>
    <li>gnomeExtensions.sleep-through-notifications</li>
    <li>gnomeExtensions.smile-complementary-extension</li>
    <li>gnomeExtensions.snx-vpn-indicator</li>
    <li>gnomeExtensions.soft-brightness-plus</li>
    <li>gnomeExtensions.somafm-internet-radio</li>
    <li>gnomeExtensions.space-bar</li>
    <li>gnomeExtensions.speedinator</li>
    <li>gnomeExtensions.ssh-search-provider-reborn</li>
    <li>gnomeExtensions.steal-my-focus-window</li>
    <li>gnomeExtensions.stopwatch</li>
    <li>gnomeExtensions.streamcontroller-integration</li>
    <li>gnomeExtensions.switch-focus-type</li>
    <li>gnomeExtensions.switch-workspace</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>gnomeExtensions.system-monitor-next</li>
    <li>gnomeExtensions.system-monitor-tray-indicator</li>
    <li>gnomeExtensions.tactile</li>
    <li>gnomeExtensions.tailscale-qs</li>
    <li>gnomeExtensions.tailscale-status</li>
    <li>gnomeExtensions.task-up</li>
    <li>gnomeExtensions.task-widget</li>
    <li>gnomeExtensions.teatimer</li>
    <li>gnomeExtensions.text-scaler</li>
    <li>gnomeExtensions.thinkpad-battery-threshold</li>
    <li>gnomeExtensions.thinkpad-thermal</li>
    <li>gnomeExtensions.tiling-assistant</li>
    <li>gnomeExtensions.toggle-alacritty</li>
    <li>gnomeExtensions.toggle-workspace-span</li>
    <li>gnomeExtensions.toggler</li>
    <li>gnomeExtensions.tophat</li>
    <li>gnomeExtensions.topiconsfix</li>
    <li>gnomeExtensions.totp</li>
    <li>gnomeExtensions.touchpad-on-off</li>
    <li>gnomeExtensions.transcodeappsearch</li>
    <li>gnomeExtensions.transmission-daemon-indicator-ng</li>
    <li>gnomeExtensions.transparent-top-bar</li>
    <li>gnomeExtensions.transparent-top-bar-adjustable-transparency</li>
    <li>gnomeExtensions.transparent-window-moving</li>
    <li>gnomeExtensions.tray-icons-reloaded</li>
    <li>gnomeExtensions.trimmer</li>
    <li>gnomeExtensions.unblank</li>
    <li>gnomeExtensions.undecorate</li>
    <li>gnomeExtensions.unlock-dialog-background</li>
    <li>gnomeExtensions.upower-battery</li>
    <li>gnomeExtensions.user-avatar-in-quick-settings</li>
    <li>gnomeExtensions.user-id-in-top-panel</li>
    <li>gnomeExtensions.user-themes</li>
    <li>gnomeExtensions.user-themes-x</li>
    <li>gnomeExtensions.utc-clock</li>
    <li>gnomeExtensions.utcclock</li>
    <li>gnomeExtensions.vertical-workspaces</li>
    <li>gnomeExtensions.vitals</li>
    <li>gnomeExtensions.voluble</li>
    <li>gnomeExtensions.volume-scroller-2</li>
    <li>gnomeExtensions.wayland-or-x11</li>
    <li>gnomeExtensions.weather-oclock</li>
    <li>gnomeExtensions.weather-or-not</li>
    <li>gnomeExtensions.week-start-modifier</li>
    <li>gnomeExtensions.weeks-start-on-monday-again</li>
    <li>gnomeExtensions.wiggle</li>
    <li>gnomeExtensions.window-calls</li>
    <li>gnomeExtensions.window-list</li>
    <li>gnomeExtensions.window-title-is-back</li>
    <li>gnomeExtensions.windownavigator</li>
    <li>gnomeExtensions.windowswitcher</li>
    <li>gnomeExtensions.wireguard-vpn-extension</li>
    <li>gnomeExtensions.workspace-indicator</li>
    <li>gnomeExtensions.workspace-switcher-manager</li>
    <li>gnomeExtensions.workspaces-indicator-by-open-apps</li>
    <li>gnomeExtensions.wsp-windows-search-provider</li>
    <li>gnomeExtensions.wtmb-window-thumbnails</li>
    <li>gnomeExtensions.x11-gestures</li>
    <li>gnomeExtensions.xazantimes</li>
    <li>gnomeExtensions.xremap</li>
    <li>gnomeExtensions.xwayland-indicator</li>
    <li>gnomeExtensions.zen</li>
  </ul>
</details>

SuperSandro2000 commented on pkgs/desktops/gnome/extensions/argos/default.nix:
```diff
@@ -1,14 +1,19 @@
+
+# There are no tagged releases anymore so we're using unstable
+# but having this keep getting updated for every upstream commit
+# seems pointless.
```
It doesn't update for every commit and has a delay of IIRC a week or so

msfjarvis replied:
Removed this part from both argos and pop-shell packages 👍🏼

SuperSandro2000 commented on pkgs/desktops/gnome/extensions/argos/default.nix:
```diff
@@ -1,14 +1,19 @@
     repo = "argos";
-    rev = "adfaa31e8c08f7b59e9492891a7e6f753c29b35e";  # https://github.com/p-e-w/argos/pull/150
-    hash = "sha256-st8AeMRtkvM4M/Z70qopjw9Yx0t9l0DsUke4ClQtcBU=";
+    rev = "0449229e11bc2bb5c66e6f1d8503635cdf276bcf";  # https://github.com/p-e-w/argos/pull/150
```
The linked PR got merged

msfjarvis replied:
All the relevant PRs are merged but there's no tagged release, I've updated the link to the GNOME 46 PR instead of the 45 one but if the preference is to just drop the link please let me know.

jtojnar replied:
I believe that was done because the commit hash pointed to an unmerged PR.

msfjarvis replied:
Removed the link.

timbertson commented on pkgs/desktops/gnome/extensions/impatience/default.nix:
```diff
@@ -15,6 +15,14 @@ stdenv.mkDerivation rec {
+      hash = "sha256-lyMQjTWJqr+JCQvKeMBC+IV5FH4aLSK61LBYQTo5xuU=";
+    })
+  ];
+
```
Sorry there was no tagged version, I've pushed `version-0.5.1` now so you can use that instead.

msfjarvis replied:
Thank you! I've dropped the commit adding the patch and bumped to 0.5.1.

msfjarvis committed:
gnomeExtensions.impatience: 0.5.0 -> 0.5.1

Diff: https://github.com/timbertson/gnome-shell-impatience/compare/refs/tags/version-0.5.0...version-0.5.1

amaxine commented:
Looks good, works, thanks!

Minor nit, f5dab0c22e69b59a98858c6c28a890a3940dc5a6 and 9cdb5272e1c6505abe4a042646679d9e7513a3e5 should also be prefixed with `gnomeExtensions`, not `gnome-extensions`.

msfjarvis commented:
Rebased the branch to fix the commit messages as suggested, there are no other changes: https://github.com/NixOS/nixpkgs/compare/43751dc94b7277de3436794081dcee491bc69caf..7ef9b3fcaeba9668e33ba49886ad299e4edac5e4
